src/cli/argument-parser.js

```diff
@@ -143,7 +132,7 @@ export default class CLIArgumentParser {
             .usage('[options] <comma-separated-browser-list> <file-or-glob ...>')
             .description(CLIArgumentParser._getDescription())
 
-            .option('-b, --list-browsers', 'output the available browser aliases')
+            .option('-b, --list-browsers [provider]', 'output available local browser aliases or the aliases for the specified browser provider')
```

Comment:
We need to change `DESCRIPTION` text as well for new `remote` syntax. ping @VasilyStrelyaev